### DISCUSSION TITLE
rabbitmq-prometheus plugin problem

### DISCUSSION THREAD

**Discussion by cichy28** [rabbitmq/rabbitmq-server#10362] [Questions] [answered]
Hi
I have installed RabbitMq 3.9.13 with rabbitmq-prometheus plugin and it looks like everything is fine. I can fetch data witch 
GET http://localhost:15692/metrics
But if i try to do it with my Prometheus which i set up in docker container it looks like this:

![image](https://github.com/rabbitmq/rabbitmq-server/assets/55942431/1cdf0351-2f8b-41e0-99d8-fb1a54d74589)

I did small investigation, and it looks like the problem is in Accept header. Prometheus send request with:

application/openmetrics-text;version=1.0.0;q=0.5,application/openmetrics-text;version=0.0.1;q=0.4,text/plain;version=0.0.4;q=0.3,*/*;q=2

And plugin can not answer for such request.

Prometheus does not allow changing request headers, and it looks like this is part of the standard.

Does anyone have such problem? Any help will be appreciated

**Comment by michaelklishin** [comment, marked as answer]
RabbitMQ 3.9 has reached [complete end of life](https://rabbitmq.com/versions.html) six month ago.
